projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/cell.py

```diff
@@ -12,21 +12,3 @@ class Cell:
     def __init__(self, jupyter_cell):
         self.tags = jupyter_cell["metadata"].get("tags", {})
         self.source = "".join(jupyter_cell["source"])
-
-
-# TODO: add a better parsing methods
-class CellUtils:
-    @staticmethod
-    def is_vdk_cell(cell: Cell):
-        return "vdk" in cell.tags
-
-    @staticmethod
-    def is_sql_cell(cell: Cell):
-        return cell.source.startswith("%sql")
-
-    @staticmethod
-    def get_cell_code(cell: Cell):
-        if cell.source.startswith("%sql"):
-            code = cell.source.replace("%sql", "")
```

Comment:
nit: if we still plan to have %sql in foreseeable future, we could keep CellUtils and link to VEP's explanation on it; otherwise we need to go back in git log and cherry-pick to bring it back, that's also a way to do it

Reply:
I don't know. I kinda feel it's better to delete it now. Code that is not used is better off deleted in most cases, right?

In the future it may be an entirely different solution. I am thinking of potential integration with SQLAlchemy for example 
IPythonSQL supports providing connection string using [SQLAlchemy URL](http://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls) standard. 

Alternatively, we might just be able to simply set teh db connection directly 
(https://github.com/catherinedevlin/ipython-sql/blob/master/src/sql/connection.py#L60)

Reply:
well will the %sql convenient interface change, or should the backing implementation rather change?
In case we want to keep the %sql user interface, then the code could match the VEP description.